src/pool/srv_layout.h

```diff
@@ -26,10 +26,10 @@
 #include <daos_types.h>
 
 /* Default layout version */
-#define DS_POOL_MD_VERSION 1
+#define DS_POOL_MD_VERSION 2
 
 /* Lowest compatible layout version */
-#define DS_POOL_MD_VERSION_LOW 1
+#define DS_POOL_MD_VERSION_LOW 2
```

Comment:
Has Johann or Liang responded to your mail on layout compatibility?

Reply:
No. Actually I forgot that I had incremented this _LOW constant in the patch. Perhaps I can restore it to 1 for the time being?